src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -0,0 +1,184 @@
+    .format('MM/DD/YYYY'),
+  optionTime3: 'PM',
+  facility: {
+    ...getVARequestMock().attributes.facility,
+    facilityCode: '983GC',
+  },
+  bestTimetoCall: ['Morning'],
+  purposeOfVisit: 'New Issue',
+  email: 'patient.test@va.gov',
+  phoneNumber: '(703) 652-0000',
+  friendlyLocationName: 'Some facility name',
+};
+
+const pending = transformPendingAppointments([appointment]);
+
+const facility = {
+  ...getVAFacilityMock().attributes,
+  id: 'vha_442GC',
+  uniqueId: '442GC',
+  name: 'Cheyenne VA Medical Center',
+  address: {
+    physical: {
+      zip: '82001-5356',
+      city: 'Cheyenne',
+      state: 'WY',
+      address1: '2360 East Pershing Boulevard',
+    },
+  },
+  phone: {
+    main: '307-778-7550',
+  },
+};
+const var983GC = transformFacility(facility);
+
+const initialState = {
+  appointments: {
+    appointmentDetails: {},
+    appointmentDetailsStatus: FETCH_STATUS.notStarted,
+    pendingStatus: FETCH_STATUS.succeeded,
+    pending,
+    facilityData: {
+      var983GC,
+    },
+  },
+};
+
+describe('VAOS <RequestedAppointmentDetailsPage>', () => {
+  beforeEach(() => {
+    mockFetch();
+    const message = getMessageMock();
+    message.attributes = {
+      ...message.attributes,
+      messageText: 'A message from the patient',
+    };
+
+    setFetchJSONResponse(
+      global.fetch.withArgs(
+        `${environment.API_URL}/vaos/v0/appointment_requests/1234/messages`,
+      ),
+      { data: [message] },
+    );
+  });
+  afterEach(() => resetFetch());
+
+  it('should render VA request details', async () => {
+    const { findByText, baseElement } = renderWithStoreAndRouter(
+      <Route path="/request/:id">
+        <RequestedAppointmentDetailsPage />
+      </Route>,
+      {
+        initialState,
+        reducers,
+        path: '/request/var1234',
+      },
+    );
+
+    expect(await findByText('Pending primary care appointment')).to.be.ok;
+    expect(baseElement).to.contain.text('VA Appointment');
+    expect(baseElement).to.contain.text('Cheyenne VA Medical Center');
+    expect(baseElement).to.contain.text('2360 East Pershing Boulevard');
+    expect(baseElement).to.contain.text('Cheyenne, WY 82001-5356');
+    expect(baseElement).to.contain.text('Main phone:');
+    expect(baseElement).to.contain.text('307-778-7550');
+    expect(baseElement).to.contain.text('Preferred date and time');
+    expect(baseElement).to.contain.text(
+      `${moment(appointment.optionDate1).format(
+        'ddd, MMMM D, YYYY',
+      )} in the morning`,
+    );
+    expect(baseElement).to.contain.text(
+      `${moment(appointment.optionDate2).format(
+        'ddd, MMMM D, YYYY',
+      )} in the morning`,
+    );
+    expect(baseElement).to.contain.text(
+      `${moment(appointment.optionDate3).format(
+        'ddd, MMMM D, YYYY',
+      )} in the afternoon`,
+    );
+    expect(baseElement).to.contain.text('New issue');
+
+    expect(await findByText(/A message from the patient/i)).to.be.ok;
```

Comment:
This test is flaky for some reason. Need to figure out what's going on.

Reply:
Disabling this check for now.  Thinking we can revisit once we work on the ticket to fetch all the data if navigating to the page directly.